src/app/core/component/graph-display/graph-display.component.ts

```diff
@@ -188,11 +226,17 @@ export class GraphDisplayComponent implements AfterContentInit, OnChanges {
     });
   }
 
+  edgeRemoved(event: CustomEvent) {
```

Comment:
A function does something, i.e., a name should indicate this. Although this is an event listener it is much more readable of the name indicates what's going on: e.g., handleEdgeRemovedEvent might be better.

Reply:
Done